fixtures/american-dj/mega-tripar-profile.json

```diff
@@ -447,7 +447,7 @@
     },
     {
       "name": "5-channel",
-      "shortName": "CH.05",
+      "shortName": "Ch.05",
```

Comment:
Hmm, maybe renaming a mode should indeed update the `lastModifyDate`...? I guess it could break some lighting software that relies on those or the export file name. So I'd rather actually change it here.

Reply:
Interesting. That one must have been a typo in the production instance and I didn't notice when I reverted the shortnames. I'll update the modify date for that one